### PR TITLE
feat: Feat poll auto options

### PR DESCRIPTION
**Pull request by OneScreenSolutions**
### What does this PR do?
This helps teacher to create poll in more convenient and quick way. From pre-compose single text it can auto formate and separate the question and it’s answers.

- Pasting all together is fast and convenient, nobody waits. Many teachers are not computer nerds, they type or use mouse slowly. Here is how this option can be enabled.

AutoOptioningOnCopiedText

- See, it can be used as far more convenient tool to update the options.

AutoOptionGenerationAndDeletion

### Motivation
Existing polling is good for its own purposes but has some disadvantages for schooling or teaching. See issue #10771
 1. Typing in some extended options is time consuming, all are waiting till that time.
 2. Targeting each field with mouse and copy and paste takes additional 10-15 seconds. 
 3. All copy and paste are real pain, when you use graphic tablet for drawing as mouse, need to grab mouse.
 4. Many sleepy and not attentive participants wake up from popup sound and after being insufficiently attentive.
 5. Keep re-asking what was the question or you keep repeating question many times if you have crowd of people. 
 6. Modal window covers the question (even if you predicted it on slide).
 7. Sometimes, during the lecture for another group of students you find, that prepared slide with question is too elementary for this group, so you skip it, making bad experience with lecture.

Further enhancement
This module can be killing feature for teaching & assessment control if percentage of correct answers at the end of lesson will be shown with ability tor teacher to export it to txt file. Mixing of options is also to be considered, at least it will make it a little harder to cheat. Of course, questions without correct options (those are just for interactions and understanding if students need more explanation) do not count, and, questions perhaps should be with one correct option for quicker development.
Example:
```
Questions asked in conference: 34
Room name: Sea pollution
Conference started: 13:05 12 May 2019 

Participants:
Ronald Decosta - 25%
Michael Brown - 78%
Ray Walker - 89%
Gilbert Mayes - 50%
```

### More

- [ ] Added/updated documentation

### CODE REVIEW

**Comment by peererror** [comment]
@antobinary yes singed and sent ..

**Comment by OneScreenSolutions** [comment]
@antobinary yes singed and sent ..

**Comment by ffdixon** [comment]
> @antobinary yes singed and sent ..

Could you resend to the email in the [docs](https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code).  Thanks!

**Comment by OneScreenSolutions** [comment]
@ffdixon done please check

**Comment by antobinary** [comment]
Thank you @OneScreenSolutions ! Confirmed

File: bigbluebutton-html5/imports/ui/components/poll/component.jsx [hunk 1]
```diff
@@ -235,13 +287,13 @@ class Poll extends Component {
...
 
   componentDidUpdate() {
-    const { amIPresenter, layoutContextDispatch, sidebarContentPanel } = this.props;
+    const { amIPresenter, layoutContextDispatch } = this.props;
 
     if (Session.equals('resetPollPanel', true)) {
       this.handleBackClick();
     }
 
-    if (!amIPresenter && sidebarContentPanel === PANELS.POLL) {
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/ui/components/poll/component.jsx:244]: Removing these parts brings back the issue fixed by #14175

**Comment by ramonlsouza** [comment]
Nice PR! Makes starting pre-made polls easier and faster
A bit similar to #11336, but not the same (perhaps both could be merged into a single feature in the future?)

File: bigbluebutton-html5/imports/ui/components/poll/component.jsx (hunk 1 above)
Comment (OneScreenSolutions) [reply bigbluebutton-html5/imports/ui/components/poll/component.jsx]: There was bug in poll due to this lines of code  following gif show it 
polling-context-bug

**Review by ramonlsouza — APPROVED** [review]

**Comment by antobinary** [comment]
Great work :confetti_ball:

**Comment by greenycat** [comment]
Great work and mindful approach! Thank YOU! Implementation is even far better than proposed.

**Pull request merged**